posts/mpox-preparedness/index.qmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "2024 mpox outbreak: common analytics tasks and available R tools"
+author:
+  - name: "James Azam"
+    orcid: "0000-0001-5782-7330"
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-147"
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+date: "2024-01-02"
+categories: [R, Epiverse-TRACE, mpox, outbreak, outbreak-analytics, DOI]
+bibliography: index.bib
+format:
+  html: 
+    toc: true
+---
+
+There is an ongoing outbreak of mpox globally and countries are on the high alert. The Democratic Republic of Congo is so far the worst hit with a total of 7, 851 cases and 384 deaths between January 1 and May 26, 2024 [^1]. Before 2022, there were no reports of sustained mpox outbreaks in most countries of the world. However, within 1 year (Jan 1, 2022, and Jan 29, 2023), 110 countries in all six WHO Regions had reported a total of 85, 473 confirmed cases and 89 deaths [@laurenson2023description].
```

Comment:
```suggestion
There are ongoing outbreaks of mpox globally. The Democratic Republic of Congo is so far the worst hit with a total of 7,851 cases and 384 deaths reported between January 1 and May 26, 2024 [^1]. Before 2022, there were few reports of sustained mpox transmission globally. However, during the following year (Jan 1, 2022, and Jan 29, 2023), 110 countries in all six WHO Regions had reported a total of 85,473 confirmed cases and 89 deaths between them [@laurenson2023description].
```
I'd refer to 'outbreaks' in each country rather than a single global outbreak.

I don't know if the additional of a space after comma is the linting, but looks strange for numbers.

Reply:
I agree with your wording.